README.md

```diff
@@ -26,6 +26,8 @@ cd fpm
 
 ### Build and Test fpm
 
+Make sure that the developers library of `gmp` is installed (e.g. `sudo apt install libgmp-dev` on Debian-derived Linux distributions)
```

Comment:
```suggestion
Make sure that the development library of `gmp` is installed (e.g. `sudo apt install libgmp-dev` on Debian-derived Linux distributions)
```

Reply:
What is the error that you are getting? I was not aware that we depend on `gmp`. How do we depend on it?

Reply:
I see, just saw the issue #98.

Reply:
> What is the error that you are getting? I was not aware that we depend on `gmp`. How do we depend on it?

For the records, the exact error follows
```
$ stack build
Preparing to install GHC (tinfo6) to an isolated location.
This will not interfere with any system-level installation.
Downloaded ghc-tinfo6-8.6.5.                                      
rts-1.0: Warning: .:464:1: The field "hugs-options" is deprecated. hugs isn't supported anymore
Installed GHC.    
Cloning 536a5e6ffb148d0dd4e4c4b120913a6744097676 from https://github.com/kowainik/tomland.git
[1 of 2] Compiling Main             ( /home/emanuele/.stack/setup-exe-src/setup-mPHDZzAJ.hs, /home/emanuele/.stack/setup-exe-src/setup-mPHDZzAJ.o )
[2 of 2] Compiling StackSetupShim   ( /home/emanuele/.stack/setup-exe-src/setup-shim-mPHDZzAJ.hs, /home/emanuele/.stack/setup-exe-src/setup-shim-mPHDZzAJ.o )
Linking /home/emanuele/.stack/setup-exe-cache/x86_64-linux-tinfo6/tmp-Cabal-simple_mPHDZzAJ_2.4.0.1_ghc-8.6.5 ...
/usr/bin/ld.gold: error: cannot find -lgmp
/home/emanuele/.stack/programs/x86_64-linux/ghc-tinfo6-8.6.5/lib/ghc-8.6.5/integer-gmp-1.0.2.0/libHSinteger-gmp-1.0.2.0.a(Type.o):integerzmgmp_GHCziIntegerziType_zdwplusBigNatWord_info: error: undefined reference to '__gmpn_add_1'
/home/emanuele/.stack/programs/x86_64-linux/ghc-tinfo6-8.6.5/lib/ghc-8.6.5/integer-gmp-1.0.2.0/libHSinteger-gmp-1.0.2.0.a(Type.o):integerzmgmp_GHCziIntegerziType_zdwminusBigNatWord_info: error: undefined reference to '__gmpn_sub_1'
/home/emanuele/.stack/programs/x86_64-linux/ghc-tinfo6-8.6.5/lib/ghc-8.6.5/integer-gmp-1.0.2.0/libHSinteger-gmp-1.0.2.0.a(Type.o):integerzmgmp_GHCziIntegerziType_zdwtimesBigNatWord_info: error: undefined reference to '__gmpn_mul_1'
/home/emanuele/.stack/programs/x86_64-linux/ghc-tinfo6-8.6.5/lib/ghc-8.6.5/integer-gmp-1.0.2.0/libHSinteger-gmp-1.0.2.0.a(Type.o):integerzmgmp_GHCziIntegerziType_zdwgo_info: error: undefined reference to '__gmpn_add'
/home/emanuele/.stack/programs/x86_64-linux/ghc-tinfo6-8.6.5/lib/ghc-8.6.5/integer-gmp-1.0.2.0/libHSinteger-gmp-1.0.2.0.a(Type.o):integerzmgmp_GHCziIntegerziType_zdwminusBigNat_info: error: undefined reference to '__gmpn_sub'
/home/emanuele/.stack/programs/x86_64-linux/ghc-tinfo6-8.6.5/lib/ghc-8.6.5/integer-gmp-1.0.2.0/libHSinteger-gmp-1.0.2.0.a(Type.o):integerzmgmp_GHCziIntegerziType_zdwtimesBigNat_info: error: undefined reference to '__gmpn_mul'
/home/emanuele/.stack/programs/x86_64-linux/ghc-tinfo6-8.6.5/lib/ghc-8.6.5/integer-gmp-1.0.2.0/libHSinteger-gmp-1.0.2.0.a(Type.o):integerzmgmp_GHCziIntegerziType_neqIntegerzh_info: error: undefined reference to '__gmpn_cmp'
/home/emanuele/.stack/programs/x86_64-linux/ghc-tinfo6-8.6.5/lib/ghc-8.6.5/integer-gmp-1.0.2.0/libHSinteger-gmp-1.0.2.0.a(Type.o):integerzmgmp_GHCziIntegerziType_neqIntegerzh_info: error: undefined reference to '__gmpn_cmp'
/home/emanuele/.stack/programs/x86_64-linux/ghc-tinfo6-8.6.5/lib/ghc-8.6.5/integer-gmp-1.0.2.0/libHSinteger-gmp-1.0.2.0.a(Type.o):integerzmgmp_GHCziIntegerziType_eqIntegerzh_info: error: undefined reference to '__gmpn_cmp'
/home/emanuele/.stack/programs/x86_64-linux/ghc-tinfo6-8.6.5/lib/ghc-8.6.5/integer-gmp-1.0.2.0/libHSinteger-gmp-1.0.2.0.a(Type.o):integerzmgmp_GHCziIntegerziType_eqIntegerzh_info: error: undefined reference to '__gmpn_cmp'
/home/emanuele/.stack/programs/x86_64-linux/ghc-tinfo6-8.6.5/lib/ghc-8.6.5/integer-gmp-1.0.2.0/libHSinteger-gmp-1.0.2.0.a(Type.o):integerzmgmp_GHCziIntegerziType_zdwquotBigNatWord_info: error: undefined reference to '__gmpn_divrem_1'
/home/emanuele/.stack/programs/x86_64-linux/ghc-tinfo6-8.6.5/lib/ghc-8.6.5/integer-gmp-1.0.2.0/libHSinteger-gmp-1.0.2.0.a(Type.o):integerzmgmp_GHCziIntegerziType_orInteger_info: error: undefined reference to '__gmpn_sub_1'
/home/emanuele/.stack/programs/x86_64-linux/ghc-tinfo6-8.6.5/lib/ghc-8.6.5/integer-gmp-1.0.2.0/libHSinteger-gmp-1.0.2.0.a(Type.o):integerzmgmp_GHCziIntegerziType_orInteger_info: error: undefined reference to '__gmpn_sub_1'
/home/emanuele/.stack/programs/x86_64-linux/ghc-tinfo6-8.6.5/lib/ghc-8.6.5/integer-gmp-1.0.2.0/libHSinteger-gmp-1.0.2.0.a(Type.o):integerzmgmp_GHCziIntegerziType_orInteger_info: error: undefined reference to '__gmpn_sub_1'
/home/emanuele/.stack/programs/x86_64-linux/ghc-tinfo6-8.6.5/lib/ghc-8.6.5/integer-gmp-1.0.2.0/libHSinteger-gmp-1.0.2.0.a(Type.o):integerzmgmp_GHCziIntegerziType_zdwquotRemBigNat_info: error: undefined reference to '__gmpn_tdiv_qr'
/home/emanuele/.stack/programs/x86_64-linux/ghc-tinfo6-8.6.5/lib/ghc-8.6.5/integer-gmp-1.0.2.0/libHSinteger-gmp-1.0.2.0.a(Type.o):integerzmgmp_GHCziIntegerziType_quotRemInteger_info: error: undefined reference to '__gmpn_divrem_1'
/home/emanuele/.stack/programs/x86_64-linux/ghc-tinfo6-8.6.5/lib/ghc-8.6.5/integer-gmp-1.0.2.0/libHSinteger-gmp-1.0.2.0.a(Type.o):integerzmgmp_GHCziIntegerziType_quotRemInteger_info: error: undefined reference to '__gmpn_divrem_1'
/home/emanuele/.stack/programs/x86_64-linux/ghc-tinfo6-8.6.5/lib/ghc-8.6.5/integer-gmp-1.0.2.0/libHSinteger-gmp-1.0.2.0.a(Type.o):integerzmgmp_GHCziIntegerziType_quotRemInteger_info: error: undefined reference to '__gmpn_divrem_1'
/home/emanuele/.stack/programs/x86_64-linux/ghc-tinfo6-8.6.5/lib/ghc-8.6.5/integer-gmp-1.0.2.0/libHSinteger-gmp-1.0.2.0.a(Type.o):integerzmgmp_GHCziIntegerziType_remInteger_info: error: undefined reference to '__gmpn_mod_1'
/home/emanuele/.stack/programs/x86_64-linux/ghc-tinfo6-8.6.5/lib/ghc-8.6.5/integer-gmp-1.0.2.0/libHSinteger-gmp-1.0.2.0.a(Type.o):integerzmgmp_GHCziIntegerziType_remInteger_info: error: undefined reference to '__gmpn_mod_1'
/home/emanuele/.stack/programs/x86_64-linux/ghc-tinfo6-8.6.5/lib/ghc-8.6.5/integer-gmp-1.0.2.0/libHSinteger-gmp-1.0.2.0.a(wrappers.o):function integer_gmp_mpn_rshift: error: undefined reference to '__gmpn_rshift'
/home/emanuele/.stack/programs/x86_64-linux/ghc-tinfo6-8.6.5/lib/ghc-8.6.5/integer-gmp-1.0.2.0/libHSinteger-gmp-1.0.2.0.a(wrappers.o):function integer_gmp_mpn_rshift_2c: error: undefined reference to '__gmpn_rshift'
/home/emanuele/.stack/programs/x86_64-linux/ghc-tinfo6-8.6.5/lib/ghc-8.6.5/integer-gmp-1.0.2.0/libHSinteger-gmp-1.0.2.0.a(wrappers.o):function integer_gmp_mpn_lshift: error: undefined reference to '__gmpn_lshift'
/home/emanuele/.stack/programs/x86_64-linux/ghc-tinfo6-8.6.5/lib/ghc-8.6.5/integer-gmp-1.0.2.0/libHSinteger-gmp-1.0.2.0.a(wrappers.o):function integer_gmp_mpn_get_d: error: undefined reference to '__gmpz_get_d'
/home/emanuele/.stack/programs/x86_64-linux/ghc-tinfo6-8.6.5/lib/ghc-8.6.5/integer-gmp-1.0.2.0/libHSinteger-gmp-1.0.2.0.a(wrappers.o):function integer_gmp_mpn_get_d: error: undefined reference to '__gmpz_get_d_2exp'
/home/emanuele/.stack/programs/x86_64-linux/ghc-tinfo6-8.6.5/lib/ghc-8.6.5/integer-gmp-1.0.2.0/libHSinteger-gmp-1.0.2.0.a(wrappers.o):function integer_gmp_gcd_word: error: undefined reference to '__gmpn_gcd_1'
/home/emanuele/.stack/programs/x86_64-linux/ghc-tinfo6-8.6.5/lib/ghc-8.6.5/integer-gmp-1.0.2.0/libHSinteger-gmp-1.0.2.0.a(wrappers.o):function integer_gmp_mpn_gcd_1: error: undefined reference to '__gmpn_gcd_1'
/home/emanuele/.stack/programs/x86_64-linux/ghc-tinfo6-8.6.5/lib/ghc-8.6.5/integer-gmp-1.0.2.0/libHSinteger-gmp-1.0.2.0.a(wrappers.o):function integer_gmp_mpn_gcd_1: error: undefined reference to '__gmpn_gcd_1'
/home/emanuele/.stack/programs/x86_64-linux/ghc-tinfo6-8.6.5/lib/ghc-8.6.5/integer-gmp-1.0.2.0/libHSinteger-gmp-1.0.2.0.a(wrappers.o):function integer_gmp_mpn_gcd: error: undefined reference to '__gmpz_init'
/home/emanuele/.stack/programs/x86_64-linux/ghc-tinfo6-8.6.5/lib/ghc-8.6.5/integer-gmp-1.0.2.0/libHSinteger-gmp-1.0.2.0.a(wrappers.o):function integer_gmp_mpn_gcd: error: undefined reference to '__gmpz_gcd'
/home/emanuele/.stack/programs/x86_64-linux/ghc-tinfo6-8.6.5/lib/ghc-8.6.5/integer-gmp-1.0.2.0/libHSinteger-gmp-1.0.2.0.a(wrappers.o):function integer_gmp_mpn_gcd: error: undefined reference to '__gmpz_clear'
/home/emanuele/.stack/programs/x86_64-linux/ghc-tinfo6-8.6.5/lib/ghc-8.6.5/integer-gmp-1.0.2.0/libHSinteger-gmp-1.0.2.0.a(wrappers.o):function integer_gmp_mpn_tdiv_q: error: undefined reference to '__gmpn_tdiv_qr'
/home/emanuele/.stack/programs/x86_64-linux/ghc-tinfo6-8.6.5/lib/ghc-8.6.5/integer-gmp-1.0.2.0/libHSinteger-gmp-1.0.2.0.a(wrappers.o):function integer_gmp_mpn_tdiv_q: error: undefined reference to '__gmpn_tdiv_qr'
/home/emanuele/.stack/programs/x86_64-linux/ghc-tinfo6-8.6.5/lib/ghc-8.6.5/integer-gmp-1.0.2.0/libHSinteger-gmp-1.0.2.0.a(wrappers.o):function integer_gmp_mpn_tdiv_r: error: undefined reference to '__gmpn_tdiv_qr'
/home/emanuele/.stack/programs/x86_64-linux/ghc-tinfo6-8.6.5/lib/ghc-8.6.5/integer-gmp-1.0.2.0/libHSinteger-gmp-1.0.2.0.a(wrappers.o):function integer_gmp_mpn_and_n: error: undefined reference to '__gmpn_and_n'
/home/emanuele/.stack/programs/x86_64-linux/ghc-tinfo6-8.6.5/lib/ghc-8.6.5/integer-gmp-1.0.2.0/libHSinteger-gmp-1.0.2.0.a(wrappers.o):function integer_gmp_mpn_andn_n: error: undefined reference to '__gmpn_andn_n'
/home/emanuele/.stack/programs/x86_64-linux/ghc-tinfo6-8.6.5/lib/ghc-8.6.5/integer-gmp-1.0.2.0/libHSinteger-gmp-1.0.2.0.a(wrappers.o):function integer_gmp_mpn_ior_n: error: undefined reference to '__gmpn_ior_n'
collect2: error: ld returned 1 exit status
`gcc' failed in phase `Linker'. (Exit code: 1)

--  While building simple Setup.hs using:
      /home/emanuele/.stack/programs/x86_64-linux/ghc-tinfo6-8.6.5/bin/ghc-8.6.5 -rtsopts -threaded -clear-package-db -global-package-db -hide-all-packages -package base -main-is StackSetupShim.mainOverride -package Cabal-2.4.0.1 /home/emanuele/.stack/setup-exe-src/setup-mPHDZzAJ.hs /home/emanuele/.stack/setup-exe-src/setup-shim-mPHDZzAJ.hs -o /home/emanuele/.stack/setup-exe-cache/x86_64-linux-tinfo6/tmp-Cabal-simple_mPHDZzAJ_2.4.0.1_ghc-8.6.5
    Process exited with code: ExitFailure 1
```
As said, installing `libgmp-dev` solved the issue.